scripts/image_tag.sh

```diff
@@ -60,6 +68,10 @@ if [[ "$version" == "latest" ]]; then
 	tag="latest"
 fi
 
+if [[ "$channel" != "" ]]; then
```

Comment:
Instead of adding a new param `--channel` I think it would just be better to add `"mainline"` and `"stable"` or whatever to the if statement on line 67:

```bash
if [[ "$version" == "latest" ]] || [[ "$version" == "mainline" ]] || [[ "$version" == "stable" ]]; then
	tag="$version"
fi
```

Then update the description at the top of the file